ereolen_support/ereolen_support.install

```diff
@@ -0,0 +1,197 @@
+<?php
+/**
+ * @file
+ * Prepare site for ereolen support module.
+ */
+
+/**
+ * Setup terms and vocabularies during install.
+ */
+function ereolen_support_install() {
+  if(empty(taxonomy_vocabulary_machine_name_load('ereolen_support_product'))) {
+    // Define product list.
+    $product_terms =[
+      'eReolen app'
+    ];
+    $vocabulary = new stdClass();
+    $vocabulary->name = 'Ereolen support form products';
+    $vocabulary->machine_name = "ereolen_support_product";
+    $vocabulary->description = t('List of products used in support form');
+    $vocabulary->module = 'taxonomy';
+
+    taxonomy_vocabulary_save($vocabulary);
+    $vid = $vocabulary->vid;
+    _ereolen_support_create_terms($product_terms, $vid);
+  }
+
+  if(empty(taxonomy_vocabulary_machine_name_load('ereolen_support_problem'))) {
+    // Define problem list.
+    $problem_terms = [
```

Comment:
Nice-to-have: If these values are moved into and function that return them it will be easier to create update hooks later using _ereolen_support_create_terms an loop over the values.

Reply:
Saved for later